modules/apps/commerce/commerce-api/src/main/java/com/liferay/commerce/constants/CommerceTaxScreenNavigationConstants.java

```diff
@@ -22,6 +22,9 @@ public class CommerceTaxScreenNavigationConstants {
 	public static final String CATEGORY_KEY_COMMERCE_REMOTE_CONFIGURATION =
 		"remote-configuration";
 
+	public static final String CATEGORY_KEY_COMMERCE_TAX_METHOD_AVALARA =
+		"settings";
```

Comment:
Hey @daiazius, the constant name does not seem right. It should be `CATEGORY_KEY_COMMERCE_SETTINGS`. Rename it please.